src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3137,7 +3137,7 @@ impl Coordinator {
                     .as_ref()
                     .expect("known to be `Ok` from `is_ok()` call above")
                 {
-                    if diff < &1 {
+                    if diff.is_negative() {
```

Comment:
@antiguru Is this the same behavior for diff == 0? @antiguru

Reply:
No, that's wrong! I'll send a PR to fix it. This can cause some rows with diff 0 to survive, but tracing the path of the data further down, it looks as if we're consolidating before writing, so we should remove the rows with the zero diff later on. Still, good to fix!